src/components/NewMeasure/CreateNewMeasureDialog.tsx

```diff
@@ -369,22 +369,22 @@ const CreateNewMeasureDialog = ({ open, onClose }) => {
                 onChange={(startDate) => {
                   formik.setFieldValue("measurementPeriodStart", startDate);
                 }}
-                renderInput={(params) => (
-                  <TextField
-                    {...params}
-                    required
-                    helperText={formikErrorHandler(
-                      "measurementPeriodStart",
-                      true
-                    )}
-                    error={
-                      formik.touched.measurementPeriodStart &&
-                      Boolean(formik.errors.measurementPeriodStart)
-                    }
-                    {...formik.getFieldProps("measurementPeriodStart")}
-                    data-testid="measurement-period-start"
-                  />
-                )}
+                renderInput={(params) => {
+                  const { onChange, ...formikFieldProps } =
+                    formik.getFieldProps("measurementPeriodStart");
+                  return (
+                    <TextField
+                      {...formikFieldProps}
+                      {...params}
+                      required
+                      data-testid="measurement-period-start"
+                      helperText={formikErrorHandler(
+                        "measurementPeriodStart",
+                        true
+                      )}
+                    />
+                  );
```

Comment:
Apparently, we don't have any test suite for the CreateNewMeasure component, Can we create a test suite and add these specific test cases, I believe we need another Tech debt story to write all test scenarios.

Reply:
I have added the test cases